src/components/layout/layout.js

```diff
@@ -34,11 +35,20 @@ const Main = styled.main`
 `
 
 const has_dataLayer = isBrowser() && window.dataLayer
-const cookie_expires = 7
+
+const cookie_domain = isProduction()
+    ? deriv_cookie_domain
+    : isBrowser() && `.${window.location.hostname}`
```

Comment:
```suggestion
    : isBrowser() && `${window.location.hostname}`
```

Reply:
im afraid, it won't work on test links or staging,  maybe we can do this instead
```
const is_test_links = hostname.includes('binary.sx')
const cookie_domain =  is_test_links ? 'binary.sx' : deriv_cookie_domain
```

// binary.sx is domain for test links and translation staging
// to run this on localhost, port forward from 8000 to 80 please follow this steps (will add it on readme)
```
sudo ifconfig lo0 10.0.0.1 alias
echo "rdr pass on lo0 inet proto tcp from any to 10.0.0.1 port 80 -> 127.0.0.1 port 8000" | sudo pfctl -ef -
```

Then edit your /etc/hosts or /private/etc/hosts file and add the following line to map your domain to 10.0.0.1.
```
10.0.0.1 static.localhost.binary.sx
```

After you save your hosts file flush your local DNS:
```
sudo dscacheutil -flushcache
```